src/jsdoc.ts

```diff
@@ -281,11 +369,12 @@ function serialize(
   if (tags.length === 0) return '';
   if (tags.length === 1) {
     const tag = tags[0];
-    if ((tag.tagName === 'type' || tag.tagName === 'nocollapse') &&
+    if ((tag.tagName === 'type' || tag.tagName === 'typedef' || tag.tagName === 'nocollapse') &&
```

Comment:
Can submit this one separately.

Reply:
I had to add this because we now use this code path to emit typedefs, and wanted to keep the text change small.